fe1-web/parts/lao/organizer/eventCreation/events/CreateElection.tsx

```diff
@@ -0,0 +1,158 @@
+  const buttonsVisibility: boolean = (electionQuestion !== '' && electionName !== ''
+    && electionBallots.length >= 2 && electionBallots[0] !== '' && electionBallots[1] !== '');
+
+  // Makes sure you can't remove ballots when there are only 2 options
+  const removeButtonVisibility: boolean = (electionBallotCounter > 2);
+
+  const onAddBallotPress = () => {
+    setElectionBallotCounter((prevCount) => prevCount + 1);
+  };
+
+  const onRemoveBallotPress = () => {
+    // decrements counter
+    setElectionBallotCounter((prevCount) => prevCount - 1);
+    // removes value from ballot array
+    const newArr = [...electionBallots];
+    newArr.splice(electionBallotCounter - 1, 1);
+    setElectionBallots(newArr);
+  };
+
+  // Updates the array with the specified ballot entries in the textfields
+  const updateBallotArray = (index: number, text: string) => {
+    const newArr = [...electionBallots];
+    newArr[index] = text;
+    setElectionBallots(newArr);
+  };
+
+  // Creates all the Ballot option text-fields based on the electionBallotCounter
+  for (let i = 0; i < electionBallotCounter; i += 1) {
+    const ballotPlaceholder = `${STRINGS.election_create_ballot_option} ${i + 1}`;
+    ballotOptionsUIComponents.push(<TextInput
+      style={styles.textInput}
+      placeholder={ballotPlaceholder} // Add index in the setState
+      onChangeText={(text: string) => { updateBallotArray(i, text); }}
+      key={ballotPlaceholder}
+    />);
+  }
+
+  const onConfirmPress = () => {
+    // Todo: Make this button send the request to create this election and remove console logs
+    console.log('Election name: ', electionName);
+    console.log('Start Date: ', startDate);
+    console.log('End Date: ', endDate);
+    console.log('Question: ', electionQuestion);
+    console.log('Voting Method: ', votingMethod);
+    console.log('Ballots: ', electionBallots);
+  };
+
+  return (
+    <ScrollView>
+      <TextBlock text={STRINGS.election_create_setup} bold />
+      <TextInput
+        style={styles.textInput}
+        placeholder={STRINGS.election_create_name}
+        onChangeText={(text: string) => { setElectionName(text); }}
+      />
+      { /* see archive branches for date picker used for native apps */ }
+      { Platform.OS === 'web' && buildDatePickerWeb() }
+      <TextBlock text={STRINGS.election_voting_method} />
+      <DropdownSelector
+        values={votingMethods}
+        onChange={(method: string) => setVotingMethod(method)}
+      />
+      <TextInput
+        style={styles.textInput}
+        placeholder={STRINGS.election_create_question}
+        onChangeText={(text: string) => { setElectionQuestion(text); }}
+      />
+      <TextBlock text={STRINGS.election_create_ballot_options} />
+      <WideButtonView
+        onPress={onAddBallotPress}
+        title="+"
+      />
+      <WideButtonView
+        onPress={onRemoveBallotPress}
+        disabled={!removeButtonVisibility}
+        title="-"
+      />
```

Comment:
this could be removed if one **remove button** is done per ballot option

Reply:
I agree with @CaiMusso , it might make sense to have a tiny button next to each ballot option, to remove any of them. This would also address @RaulinN 's remark

Reply:
This is implemented